docs/README.md

```diff
@@ -30,28 +30,34 @@ locally and review your changes, you will need to do the following:
 2. Install Julia by following [the relevant
    guide](https://julialang.org/downloads/).
 
-3. Navigate in a terminal to `Turing.jl/docs/`. Run `julia make.jl`,
+3. Clone the [`turing.ml` repository](https://github.com/TuringLang/turing.ml), 
+   where the website is built from.
+
+4. Add your local version of `Turing.jl` to the `turing.ml` environment by running
+   `using Pkg; Pkg.dev("path/to/Turing.jl")`.
```

Comment:
I don't understand this instruction, what do you have in mind? Where's the `turing.ml` environment? It seems you just add Turing.jl to your global environment since no environment was activated and you only navigate to turing.ml in the next step?

I guess also commonly one would run `] dev Turing` as initial command to check out and work on the Turing repo. Then it will be cloned to .julia/dev/Turing automatically.

Reply:
When testing, I activated the `turing.ml` environment, by going into the repo and running `] activate .` This is what I usually do when working on packages. Also, the first two lines in the `make.jl` file say `import Pkg; Pkg.activate(".")`, so I thought it would be a more appropriate test. Do you think this is unnecessary? 

I forked the repo and then added it by running `] dev .` when being in the repo. However, then the `.julia/dev/` folder was empty. Anyways, I assume the instructions could be simpler so I am open to any suggestions.

Reply:
To avoid confusion and make it clearer that one has to use turing.ml, maybe we could remove all instructions and files other than the md sources, and only add a note that links to the instructions in the turing.ml repo?

Of course, then the question is still how to set it up and explain it in the turing.ml repo but we would reduce redundancies, inconsistencies, and possibly avoid some confusion.

Reply:
Yes I think this is the way to go. I'll add a commit to this PR later today.